recipes/gravelamps/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "gravelamps" %}
+{% set version = "2.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/mick.wright/Gravelamps/-/archive/{{ version }}/Gravelamps-{{ version }}.tar.gz
+  sha256: 57162d910f1c56a95f99dd7bec19d9cf0fc80469ad12b255aed6059fe91c5cbf
+
+build:
+  entry_points:
+    - gravelamps_inference=gravelamps.inference:main
+    - gravelamps_generate_lens=gravelamps.generate_lens:main
+    - gravelamps_generate_interpolator_data=gravelamps.lensing.generic:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools-scm
+    - pip
+    - make
+    - arb
+    - libflint
+    - libgomp
+  run:
+    - python
+    - asimov
+    - astropy
+    - bilby
+    - bilby_pipe
+    - python-htcondor
+    - numpy
+    - scipy
+    - make
+    - arb
+    - libflint
+    - libgomp
+    - {{ compiler('cxx') }}
```

Comment:
Is a compiler actually needed at *run* time?

Reply:
Oh no, apologies. Just during Install, that's been fixed now.